frontend/src/components/details/person.svelte

```diff
@@ -1,14 +1,16 @@
 <script lang="ts">
-  import { mediaIdToUrlConverter } from "../../utils"
+  import { mediaIdToUrlConverter, calculateAmountOfShownItems } from "../../utils"
   import type { Cast } from "../../types"
 
   export let cast: Cast[]
 
   const LOW_RES_IMG_URL = "https://image.tmdb.org/t/p/w500/"
-  const SHOW_BUTTON_AMOUNT = 18
-  const CAST_ITEM_START_AMOUNT = 9
 
-  let castItemAmount = 9
+  let castItemAmount = calculateAmountOfShownItems(
+    window.visualViewport.width,
+    [27, 24, 21, 15, 12, 9]
```

Comment:
How would it look with a row less on the bigger resolutions? 27 faces is a bit overwhelming

![image](https://user-images.githubusercontent.com/43907402/155806092-ae75d9ed-7e92-44d2-8613-20a3ca3d696f.png)